ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/fabric/interfaces/super-spine-p2p-interfaces.j2

```diff
@@ -24,6 +24,10 @@
     mtu: {{ p2p_uplinks_mtu }}
 {%             if p2p_uplinks_qos_profile is arista.avd.defined %}
     service_profile: {{ p2p_uplinks_qos_profile }}
+{%             endif %}
+{%             if p2p_uplinks_ptp_enable is arista.avd.defined %}
+    ptp:
+        enable: {{ p2p_uplinks_ptp_enable }}
```

Comment:
only 2-space indentation in "structured configuration"

Reply:
Changed, thanks!